arcaflow_plugin_uperf/uperf_schema.py

```diff
@@ -131,6 +124,14 @@ class ProfileFlowOpConnection(ProfileFlowOpCommon):
             "description": "The SSL Engine used by OpenSSL",
         },
     )
+    port: int = field(
+        default=20001,
+        metadata={
+            "name": "port",
+            "description": "The port used to establish the connection."
+                           " This is not the same as the comm_port.",
+        },
+    )
```

Comment:
`port` doesn't seem to be a [documented "connect" Flowop keword](https://uperf.org/manual.html#id2533709).  Does this value actually do anything?

Reply:
Everything in this PR is from experimentation as opposed to documentation. It's horribly documented. This port seems to be the one for the connection being tested. And that can be just a connection to repeatedly test opening connections, or it could be the single connection that data is transferring through for throughput tests.

Meanwhile, `comm_port` is what's just establishing orchestration communication to allow the test to work.